conda_devenv/devenv.py

```diff
@@ -151,11 +158,16 @@ def merge_dependencies_version_specifications(yaml_dict, key_to_merge):
             if m is None:
                 raise RuntimeError('The package version specification "{}" do not follow the'
                                    ' expected format.'.format(dep))
+            # Consider the channel name as part of the package name. If multiple channels are specified, the package
```

Comment:
Doesn't this lead to an unsolvable error?

If it causes the error: Not that we should somehow deal with this, just mention in the comment and in the docs.

Reply:
No, `conda env` is able to solve it, and it probably uses the channels priorities (I've tested duplicating it with and without channel spec, and also with two different channel specs) ;)

I think it's best to not repeat documentation for `conda env` behavior here, since that may change in the future.